databases/Assignment2_answers.sql

```diff
@@ -0,0 +1,316 @@
+1. Which laptop models are shown in the Product relation?
+SELECT DISTINCT(Model)
+FROM Product
+WHERE Type = laptop;
+
+2. Find the set of all manufacturers ordered alphabetically.
+SELECT DISTINCT(Manufacturer)
+FROM Product
+ORDER BY DESC; # is there an alpha order?
```

Comment:
to clarify, if i do not specify which col in the ORDER BY, it will just do it using the first col?
or do I ALWAYS have to specify a col?